proto/substrait/extended_expression.proto

```diff
@@ -19,33 +19,40 @@ message ExpressionReference {
     AggregateFunction measure = 2;
   }
   // Field names in depth-first order
+  //
+  // For primitive types, this will just be a single name. Nested types may
+  // have multiple names.
   repeated string output_names = 3;
 }
 
-// Describe a set of operations to complete.
-// For compactness sake, identifiers are normalized at the plan level.
+// Describe a list of expressions to evaluate.
```

Comment:
Perhaps either "a list of expressions to evaluate" or "describes a list of expression to evalute"?

Reply:
Yeah I prefer this, just the plan description was `Describe a set of operations to complete.` so thought it might be a protobuf convention.

```suggestion
// A list of expressions to evaluate.
```